assets/app/scripts/app.js

```diff
@@ -194,6 +195,14 @@ angular
         templateUrl: 'views/browse/service.html',
         controller: 'ServiceController'
       })
+      .when('/project/:project/browse/storage', {
+        templateUrl: 'views/storage.html',
+        controller: 'StorageController'
+      })
+      .when('/project/:project/browse/persistentvolumeclaims/:pvc', {
```

Comment:
Great Q, what i suggested is prob not the best.  Is `.when('/project/:project/attachPersistentVolumeClaim')` most fitting?  Im assuming with this path it would be being attached to the project, rather than a pod/deployment/etc.